src/time_array_container.jl

```diff
@@ -1,82 +0,0 @@
-struct TimeDataContainer <: InfrastructureSystemsType
-    data::SortedDict{Dates.DateTime, Array}
-    resolution::Dates.Period
-    internal::InfrastructureSystemsInternal
-
-    function TimeDataContainer(
-        data::SortedDict{Dates.DateTime, Array{T, N}},
-        resolution::Dates.Period,
-        internal::InfrastructureSystemsInternal,
-    ) where {T, N}
-        series_length = length(first(values(data)))
-        for (k, v) in data
```

Comment:
I understand the removal of TimeDataContainer but couldn't find where did these checks go.

Reply:
Good catch.  fixed.